flake8.cfg

```
@@ -2,7 +2,7 @@
 max-line-length=150
 ignore =
     # Ignore missing type annotation for self, *args, **kwargs (respectively)
-    ANN101, ANN001, ANN002, ANN003,
+    ANN101, ANN102, ANN001, ANN002, ANN003
```

Add a comma here, there is another error code under this
```suggestion
    ANN101, ANN102, ANN001, ANN002, ANN003,
```